docs/source/blueprints/BP000.rst

```diff
@@ -0,0 +1,319 @@
+
+ * Author: The author or authors of blueprint
+
+ * Reviewers: All reviewers that approved and helped during the review process
+
+ * Created: Date string when the blueprint first draft was submitted. Please
+   use the following format: DD-MMM-YYYY.
+
+ * Type: One of the types described during the previous section
+
+ * Status: One of the types described during the next section
+
+Here is an example of a header::
+
+   :Number: BP001
+   :Title: Configuration by convention
+   :Author: Beraldo Leal <bleal@redhat.com>
+   :Reviewers: Cleber Rosa, Lukáš Doktor and Plamen Dimitrov
+   :Created: 06-Dec-2019
+   :Type: Epic Blueprint
+   :Status: WIP
+
+Blueprint statuses
+------------------
+
+ * Draft: All blueprints should be created in this state. This means the
+   blueprint is accepting comments, and probably there is a discussion
+   happening. Blueprints in draft mode can be part of our repository.
+
+ * Approved: Blueprint was approved after discussions, and all suggestions are
+   already incorporated on the document. Nobody has started working on this
+   yet.
+
+ * Assigned: This status is not about the blueprint itself, but about the proposal
+   that is the subject of the BP. This means that the blueprint was approved,
+   and someone is already working on implementing it. A BP status can change
+   from Draft to Assigned if the work has started already.
+
+ * Implemented: This means the BP is already implemented and delivered to the
+   Avocado’s master branch.
+
+ * Rejected: Rejected status means the idea was not implemented because it
+   wasn't approved by everyone or has some techinical limitations.
+
+ * Deprecated: Deprecated means it was approved, implemented, and at some
+   point, makes no more sense to have it. For example, anything related to the
+   legacy runner. Usually, Deprecated means that it was replaced by something
+   else.
+
+As you can see, there is no status to accommodate any future change in a
+blueprint. Blueprints should not be “voided.” Any improvement on an old
+blueprint should be presented as a new blueprint, changing the status of the
+original to “deprecated”.
+
+Sections
+--------
+
+In order to facilitate the reading and understanding of the problem, all
+blueprints must have the following sections:
+
+ * TL;DR
+
+ * Motivation
+
+ * Specification
+
+ * Backwards Compatibility
+
+ * Security Implications
+
+ * How to Teach This
+
+ * Related Issues
+
+ * References
+
+Bellow a brief description of what you should write in each section:
+
+ * TL;DR: Should be a short description of your blueprint. Like an abstract. We
+   recommend writing this at the end of your first draft. This will give you a
+   better overview of it.
+
+ * Motivation: This should be the motivation of your proposed solution, not the
+   motivation of the blueprint itself. It describes the problem. Here, the
+   proposed solution should answer “why” this is needed.
```

Comment:
Not sure why "the proposed solution should answer" and what this means. I guess you mean "Here, you should answer "why" your solution is needed."